### ISSUE TITLE
Home Screen changes for Ratio and Proportion

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#187]
Related to some minor changes in https://github.com/phetsims/joist/issues/665

In the last interview for Ratio and Proportion, it took one learner using a screen reader about 15 minutes to load an actual screen. It is expected to take longer to get to a sim screen because learners usually read a lot more before they make a decision to interact, or choose their interactive path.

While the user experienced a few technical issues including getting stuck in the address bar and reloading the page, it should be more clear in what order is the best way to explore the sim screen buttons for non-visual learners. 

In the visual design, the first screen button visually screams out, "Start Here!"

We are not screaming at the same level in the non-visual experience, so @Matthew-Moore240 and I suggest the following changes to the screen button interaction hints:

**H2: Sim Screens**
1. Discover Screen, button
    P: **_Start here and discover set ratio challenges._**
2. Create Screen, button
    P: **_Set your own ratio challenges and explore._**

**Comment by emily-phet** [comment]
@terracoda We don't always know if a learner will be experiencing the sim in the configuration we test it in. There is a publicly known query parameter that allows teacher to create a link that sets which screen(s) and in what order the learner sees the screen. So it's entirely possible that the learner gets the sim with *only* one screen, or screens in reverse order. 

I think we'll have to rely on the activity from the teacher to help direct the learner, if they're unsure where to start. We can't indicate a "first" or "second".

**Comment by emily-phet** [comment]
Or - the "start here" needs to be dynamic, and know what the screen configuration is regardless of changes.

**Comment by terracoda** [comment]
@emily-phet, re https://github.com/phetsims/ratio-and-proportion/issues/187#issuecomment-700002612, I thought using "Start here" would be problematic, I just wasn't sure why.

The words, "Start here", are less important than the wording of the actual hint for the screen. Maybe it's just the wording of the screen hint that needs to be adjusted.

**Comment by terracoda** [comment]
Maybe this is better: 
1. Discover Screen, button
    P: Explore this screen to discover set ratio challenges.
2. Create Screen, button
    P: On this screen, set your own ratio challenges to explore.

**Comment by zepumph** [comment]
Let me know when you have something you would like to see in the sim.

**Comment by terracoda** [comment]
@zepumph, let's go with content in https://github.com/phetsims/ratio-and-proportion/issues/187#issuecomment-700008069

**Comment by zepumph** [comment]
Implemented above.

**Comment by terracoda** [comment]
Looks and sounds awesome @zepumph. 
In phetsims/joist#665, you'll update the static introductory sentence to include "interactive":
- "Come explore with {{Ratio and Proportion}}. It has {{2}} **interactive** screens."

The content in this issue is verified, so closing.
